### ISSUE TITLE
indent lists properly

### ISSUE THREAD

**Issue by waldenn** [spencermountain/wtf_wikipedia#360]
This [article](https://en.wikipedia.org/wiki/List_of_regions_in_the_human_brain) uses many unordered list indentation (with some text portions mixed in also). It serves as a good testing case.

Currently wtf indents all these list items (ordered or unordered) only to 1 depth level.

If you have some hints on what places "list depth state" for the HTML is relevant it would be great. I want to take a look at how difficult it is to fix this. Not sure I can fix it, but I can give it a try :-)

**Comment by spencermountain** [comment]
yeah, list indentation is something we should support, but will be a breaking change.
adding to v9 milestone

**Comment by spencermountain** [comment]
if anyone has any good ideas about how this should work, I'm all-ears.
right now `.lists()` returns an array of sentences.
Maybe it should be `[{depth:n, sentence:s}]` or something like that.

or maybe `[{sentence:s, children:[]}]`

hoping to ship v9 this month
